Explanation on PoW finality block is 0 (#189)
JoshOrndorff/recipes: text/3-entrees/basic-pow.md

## The Light Client

The last thing in the `service.rs` file is constructing the [light client](https://www.parity.io/what-is-a-light-client/)'s service. This code is quite similar to the construction of the full service.

Instead of using the `with_import_queue` function we used previously, we use the `with_import_queue_and_fprb` function. FPRB stand for [`FinalityProofRequestBuilder`](https://substrate.dev/rustdocs/master/sc_network/config/trait.FinalityProofRequestBuilder.html). In chains with deterministic finality, light clients must request proofs of finality from full nodes. But in our chain, we do not have deterministic finality, so we can use the [`DummyFinalityProofRequestBuilder`](https://substrate.dev/rustdocs/master/sc_network/config/struct.DummyFinalityProofRequestBuilder.html) which does nothing except satisfying Rust's type checker.

Once the dummy request builder is configured, the `BlockImport` and import queue are configured exactly as they were in the full node.

> ## Other Notes

Reviewer: ```suggestion
## Note of Finality
```

> If we run the PoW node now, we will see console logs as follow, that the finalized block always remains at 0.

Reviewer: ```suggestion
If we run the `basic-pow` node now, we see in console logs, that the finalized block always remains at 0.
```
